### ISSUE TITLE
（1）关于高级API文档model.fit的参数 （2）关于bert_tokenizer 预训练参数的说明

### ISSUE THREAD

**Issue by xu-peng-7** [PaddlePaddle/Paddle#39598]
model.fit(参数)
官网只有该参数的简单使用示例，但是AIstudio中有部分项目用了比较丰富的参数，想看一下完整的参数文档，没有找到
BertTokenizer.from_pretrained('参数')
该参数部分传入的时bert分词的预训练模型名称，目前能找到的都是中文的分词模型，但是看到了有其他的，想知道该部分一共有多少的预训练模型，应该传入什么样的参数使用。文档未找到。

**Comment by pangyoki** [comment]
你好，`model.fit()`的文档可以参考：https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Model_cn.html#fit-train-data-none-eval-data-none-batch-size-1-epochs-1-eval-freq-1-log-freq-10-save-dir-none-save-freq-1-verbose-2-drop-last-false-shuffle-true-num-workers-0-callbacks-none 。
`from_pretrained`方法的描述可以看：https://github.com/PaddlePaddle/Paddle/blob/a909bdf154fa0d44124b73dcdb1c8c4205c83999/python/paddle/fluid/tests/unittests/tokenizer/tokenizer_utils.py#L386 ，里面有介绍加载预训练模型的方法。
